identity/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "chrome.identity example",
+  "version": "1",
+  "manifest_version": 2,
+  "applications": {
+    "gecko": { "id": "identity.example@mozilla.org" }
```

Comment:
Since the id changes every time you reload the temporary addon, and the redirect uri is based on the id, you kind of need a hard set id to test oauth stuff.